src/applications/personalization/view-dependents/containers/ViewDependentsApp.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 
-import ViewDependentsLayout from '../components/ViewDependentsLayout';
+import ViewDependentsLayout from '../layouts/ViewDependentsLayout';
 
 class ViewDependentsApp extends Component {
```

Comment:
Just realized that this will eventually have to be gated behind authentication, which could impact our desire to not use redux 😬